### PR TITLE
feat: add redis user and ssl

### PR DESCRIPTION
**Pull request by netanelC**

### CODE REVIEW

File: nginx-s3-gateway/example/.env [hunk 1]
```diff
@@ -17,6 +17,7 @@ NGINX_WORKER_PROCESSES=4
 # Redis
 REDIS_HOST=redis
 REDIS_PORT=6379
+REDIS_SSL_ENABLED=false
```
Comment (syncush) [inline nginx-s3-gateway/example/.env:20]: if we enable SSL, how do we provide a certificate or public key ?

**Review by syncush — CHANGES_REQUESTED** [review]

File: nginx-s3-gateway/example/.env (hunk 1 above)
Comment (netanelC) [reply nginx-s3-gateway/example/.env:20]: At this point, we don't. This is for enabling TLS connection without providing the certificates directly

Comment (syncush) [reply nginx-s3-gateway/example/.env:20]: then SSL should be always false without a parameter since we dont support `REDIS_SSL_ENABLED=true`

Comment (netanelC) [reply nginx-s3-gateway/example/.env:20]: We can support SSL connection without providing the certs if there is a third-party that provides the certs. In that case, all we need is to enable the SSL connection

**Review by syncush — APPROVED** [review]

File: nginx-s3-gateway/example/.env (hunk 1 above)
Comment (shimoncohen) [reply nginx-s3-gateway/example/.env:20]: You should still allow to provide the certs if wanted.

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**